src/oci-limits-mcp-server/requirements.txt

```diff
@@ -0,0 +1,3 @@
+# OCI Limits MCP Server Dependencies
```

Comment:
you can replace this file with an entry in pyproject.toml
https://github.com/oracle/mcp/blob/main/src/oci-api-mcp-server/pyproject.toml#L12-L15